### ISSUE TITLE
 Repeatable test condition for traffic load

### ISSUE THREAD

**Issue by billwuqin** [cpignata/powerbench#9]
3.	Section 4.2 of draft-cprigf-bmwg-pwoerbench said:
“
   The traffic load must specify packet sizes, packet rates, and inter-
   packet delays, as all may affect the energy consumption of network
   devices.
”
I believe this requirement enable repeatable test condition, which require to specify the exact repeating sequence of packet sizes from the usual set of fixed sizes
   and from other forms of mixed-size specification. However I am not sure the description in section 4.2, e.g., specify inter-packet delays,  is consistent with ETSI or ITU-T specification,
ATIS specification define IMIX traffic, which is A realistic mixture of packet sizes can be approximated by a set of packet lengths (three or more) that represent 
the common modal lengths, plus an even distribution of every other packet size.
One example of IMIX model is:
Table 2: Simple IMIX model 
Package Size (Bytes) Proportion of Total     Bandwidth
64                               7 parts                         10,3 
594                            4 parts                          54,7 
1 518                         1 part                            35

**Comment by romain-jacob** [comment]
Agreed. Ultimately we should have some very clearly defined traffic profiles and appropriate tooling built to make is easy to generate those profiles. 

We have started some work on this in my group, but it is still very preliminary.

**Comment by cpignata** [comment]
Strengthened the IMIX/RFC 6985 reference as the repeatability-satisfying traffic specification; removed inter-packet delays as a required parameter, consistent with ETSI and ATIS practice.
